libs/platform/unit/platform_unit_test.vcxproj

```diff
@@ -127,7 +127,7 @@
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
-      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
```

Comment:
Revert or add explanation to PR description

Reply:
reverted.